cou/apps/auxiliary.py

```diff
@@ -71,20 +69,21 @@ def os_origin_config(self, target: OpenStackRelease) -> Optional[OpenStackReleas
         :return: OpenStackRelease object or None if the app doesn't have os_origin config.
         :rtype: Optional[OpenStackRelease]
         """
-        os_origin_parsed = None
-        if self.os_origin is not None:
-            # Ex: "cloud:focal-ussuri" will result in "ussuri"
-            os_origin_parsed = self.os_origin.rsplit("-", maxsplit=1)[-1]
-            match os_origin_parsed:
-                case "distro":
-                    # find the OpenStack release based on ubuntu series
-                    os_origin_parsed = LTS_SERIES[self.series]
-                case "":
-                    # if it's empty we consider the previous release from the target.
-                    # Ex: rabbitmq-server has empty "source" and receive target "victoria".
-                    # In that case it will be considered as ussuri and with the upgrade,
-                    # "source" config will be changed to "cloud:focal-victoria".
-                    os_origin_parsed = target.previous_release
+        if not self.os_origin:
+            return None
+
+        # Ex: "cloud:focal-ussuri" will result in "ussuri"
+        os_origin_parsed: Optional[str] = self.os_origin.rsplit("-", maxsplit=1)[-1]
```

Comment:
not-blocking: This comment is only my preferences of how you we can extract parameters from tuple, list, etc. For me it's more readable. 
```suggestion
        *_, os_origin_parsed: Optional[str] = self.os_origin.rsplit("-", maxsplit=1)
```

Reply:
I like the proposal, but isn't the unpacking operator unnecessary since `maxsplit=1` ?